include/morphio/properties.h

```diff
@@ -158,9 +176,16 @@ struct CellLevel {
     bool diff(const CellLevel& other, LogLevel logLevel) const;
     bool operator==(const CellLevel& other) const;
     bool operator!=(const CellLevel& other) const;
-    std::string fileFormat() const;
-    uint32_t majorVersion();
-    uint32_t minorVersion();
+
+    std::string fileFormat() const {
+        return std::get<0>(_version);
```

Comment:
This change does not pass the compilation step with clang 
https://github.com/BlueBrain/MorphIO/runs/2718243922#step:5:240